ams/ams-server/src/main/java/com/netease/arctic/ams/server/service/TerminalService.java

```diff
@@ -149,7 +149,7 @@ public static SesssionInfo executeSql(String catalog, String sqls) {
         }
         try {
           sqlRunningInfo.getLogs().add(df.format(new Date()) + " use " + catalog);
-          spark.sql("use " + catalog);
+          spark.sql("use " + "`" + catalog + "`");
```

Comment:
PLS uses the `static final` keywords to define the String parameter: "`".

e.g. private static final String QUOTE = "`";

Reply:
> @shendanfengg, is there any other places should also quote the catalog?

No need to quote other place,only ams-server terminal needs this change

Reply:
> PLS uses the `static final` keywords to define the String parameter: "`".
> 
> e.g. private static final String QUOTE = "`";

It has been fixed